### DISCUSSION TITLE
WG Proposal template

### DISCUSSION THREAD

**Discussion by cescoffier** [quarkusio/quarkus#44967] [Design Discussions]
Hello,

I started drafting a template for the WG proposal. It's totally optional, but if you are struggling, it can help:
Here is the template (remember, proposals are GitHub discussions):

Title of the discussion: WG - Name of the working group
Initial message content:

**Objective**:
The main objective of this working group is to **<enter the main objective, one sentence only>**.

**The Problem**

Describe the current issue, limitation, or gap necessitating this working group. Make sure to clearly articulate the impact of this problem and why it needs to be addressed.

**The proposed Solution**

Provide an overview of the working group’s approach to address the problem. Keep this section focused on high-level strategies rather than implementation details.

**Definition of Done**

Outline clear, measurable criteria that define when this working group will be considered complete.

 Examples include:

* Delivering specific technical contributions (e.g., implementation of features, resolved issues, or merged PRs).
* Publishing ADRs or design documentation.
* Creating new or updated documentation.
* Sharing outcomes via blog posts, demos, or videos.

**Scope of Work**

Define the boundaries of the working group’s focus. Clarify what is in and **out of scope** to avoid ambiguity.

**Organizing the Work**

1. **Communication and Transparency**:

* Explain how the coordination will be done (GitHub issues, GitHub project…)
* Explain where most of the work will be done (Quarkus core, Quarkiverse…)

2. **Point of Contact**:

* Name the main point of contact for this working group

3. **Expected Timeline**:

* Estimate the timeframe for the working group’s lifecycle. If needed, break it into milestones (e.g., initial exploration, design phase, implementation phase). It may become a list of smaller working groups. 

**(Optional) Potential Outcomes & Deliverables** 

List the expected deliverables or results of this working group. 

**(Optional) Existing issues**

If there are issues already opened about the problem, list them. It will ease the board creation.

**Comment by xstefank** [comment]
A unified structure will help organize working groups, so I wouldn't object if it would be mandatory.
